components/chart/linearTimeSeries.js

```diff
@@ -0,0 +1,127 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.charts.linearTimeSeries = function() {
+
+        var xScale = d3.time.scale();
+        var yScale = d3.scale.linear();
+        var xAxis = d3.svg.axis()
+            .scale(xScale)
+            .orient('bottom');
+        var yAxis = d3.svg.axis()
+            .scale(yScale)
+            .orient('left');
```

Comment:
I was taking heavy inspiration from http://www.highcharts.com/stock/demo. Unfortunately this is something that ends up baked into the chart as it stands...